.env.cluster

```diff
@@ -72,12 +72,17 @@ HAPI_PROXY_INSTANCES=3
 # Identity Access Manager - Keycloak
 KC_FRONTEND_URL=https://keycloak.domain
 KC_GRAFANA_ROOT_URL=https://grafana.domain
+KC_JEMPI_ROOT_URL=https://jempi-web.domain
 KC_SUPERSET_ROOT_URL=https://superset.domain
 KC_OPENHIM_ROOT_URL=https://openhimconsole.domain
 GF_SERVER_DOMAIN=grafana.domain
 
 # Client Registry - JeMPI
 JEMPI_WEB_INSTANCES=3
+# Only Keycloak authentication is available in Jempi
```

Comment:
Is this dependency specified in the `packagemetadata.json`?

Reply:
the default is disabling the SSO of jempi, because we re not using that always, so for now the only authentication type avaialble is Keycloak auth, there is a normal login coming soon I think, so now we need that to be true when running keycloak and jempi

Reply:
So for this version of JeMPI and Platform, it should be a dependency as we cannot make it work without the dependency. It can be removed in future once the other functionality is implemented. It should also default to use SSO in this version.

Reply:
cool done :)